tmt/steps/discover/__init__.py

```diff
@@ -239,10 +259,10 @@ def wake(self) -> None:
     def summary(self) -> None:
         """ Give a concise summary of the discovery """
         # Summary of selected tests
-        text = listed(len(self.tests()), 'test') + ' selected'
+        text = listed(len(self.tests(enabled=True)), 'test') + ' selected'
```

Comment:
@happz how does this `enable=True` relate to the changes? Is it some fix?

Reply:
Th signature and behavior changed in https://github.com/teemtee/tmt/pull/1790/commits/7b3385b5f9a0bd393e8c2cc44241e7db73897304 as part of this effort, `tests()` must now be aware of phase name and whether to list enabled tests.